cowprotocol/cowswap
# refactor: remove affiliate section and related logic

# Summary

Fixes #2300 

Removes affiliate banners, faq section, account section.

<img width="1183" alt="Screenshot 2023-04-18 at 18 43 37" src="https://user-images.githubusercontent.com/5172699/232846758-2e7589e6-daa5-4541-b27b-bf9b13f5eac7.png">

We leave the code of these components in as we might want it in the future.

# To Test

1. Open CoW Swap
2. Go to accounts
3. Should not see affiliate section
4. Follow an affiliate link (like this https://swap.cow.fi/#/account?referral=0x9FA3c00a92Ec5f96B1Ad2527ab41B3932EFEDa58&chain=mainnet)
5. You should not see affiliate banner
6. Hover over FAQ
7. You should not see affiliate links there
8. Open affiliate page by link (like https://swap.cow.fi/#/faq/affiliate?chain=mainnet)
9. You should get 404

  # Background

*Optional: Give background information for changes you've made, that might be difficult to explain via comments*
